src/mappings/poolFactory.ts

```diff
@@ -241,6 +241,8 @@ function handleNewLinearPool(event: PoolCreated, poolType: string, poolTypeVersi
 
   let pool = handleNewPool(event, poolId, swapFee);
 
+  // do we want to set all to LINEAR type? since we also have aavelinearpools
+  // pool.poolType = PoolType.Linear
```

Comment:
mh no i think i would leave it like this

Reply:
but doesn't that mess-up things in our backend when the aave linears suddently have AAVELINEAR as type and not LINEAR?